sci-geosciences/qmapshack/qmapshack-1.17.1_p20250426-r1.ebuild

```
@@ -5,11 +5,11 @@ EAPI=8
 
 inherit cmake xdg
 
-COMMIT="948ae8a8a1526766ebdf9df2f207462c09c15c3b"
+COMMIT="71861d34a91c33380ba1ea055ff2ae67209e2bd6"
 
 DESCRIPTION="GPS mapping utility"
 HOMEPAGE="https://github.com/Maproom/qmapshack/wiki"
-SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${PF}.tar.gz"
```

I wouldn't expect a new tarball under `-r1` revbump, and this will potentially be a huge surprise on the next minor revbump fix someone wants to do on that ebuild. While we don't know how many more revisions this will take for them to get right, let's pick a page out of the kde.org playbook:

```suggestion
SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${P}-${COMMIT:0:8}.tar.gz"
```

---

next push will use another commit